spec/java/util/Optional.lsl

```diff
@@ -38,8 +77,8 @@ import "java/util/stream/_interfaces.lsl";
 
     shift Allocated -> Initialized by [
         // constructors
-        Optional (),
-        Optional (T),
+        Optional (Optional),
```

Comment:
This automaton has two constructors in opposite to real Optional class. In original class there is only one constructor.
As I understood this is needed for generation process ? Because we are using indirect generation ?